stdlib/Logging/docs/src/index.md

```diff
@@ -217,6 +217,9 @@ julia> foo()
 
 ```
 
+Use a comma separator to enable debug for multiple modules: 
```

Comment:
```suggestion
Use a comma separator to enable debug for multiple modules:
```

Reply:
Yes, I've seen that now. Will fix in 2 secs.